superset-frontend/src/explore/components/controls/MetricControl/AdhocMetricEditPopover.jsx

```diff
@@ -370,7 +374,7 @@ export default class AdhocMetricEditPopover extends React.PureComponent {
                       }
                       key={savedMetric.id}
                     >
-                      {this.renderColumnOption(savedMetric)}
+                      {this.renderSavedMetricOption(savedMetric)}
```

Comment:
Why go to the trouble of calling a function here instead of simply using the `<MetricOption>` component inline?

Reply:
Ugh, that was a leftover that I should have refactored. Thanks for pointing that out.